pages/doc/authentication.md

```diff
@@ -18,32 +18,42 @@ After a user has been successfully authenticated, administrators determine which
 
 ## Multi-Tenant Authentication
 
-Some VMware customers set up their environment to support logins into different tenants on the same cluster. In most cases, a user is either part of one tenant or part of another tenant, for example, either part of the networking team or part of the storage team. But in some cases, the same user has to have access to both tenants.
+Most existing Wavefront customers set up their environment to support logins into a single tenant on a cluster. If they want their different teams to access diffrent instances(tenants), those tenants need to be in separate clusters, as we used to allow an authenticated user to log in to a single customer. To allow multiple teams from same company to use different customer instance on the same cluster, the instances need to use different authentication providers, which often needed complex tweaks on the customer side. 
+
+However, with growing need to smoothly allow multiple teams from same company to reside as separate tenants inside a single cluster (for better utilization of a cluster), we are now introducing the multi-tenancy feature into Wavefront - starting with release 2018.40. With this feature, different teams under the same company can coexist as different teantns in the same cluster and no need to tweak the identity providers on the customer side. Even more, a single user from the company can be part of multiple tenants and they can smoothly switch between their accounts in the different teams.
 
 For that use case:
 * Administrators invite users who belong to one team to one tenant
 * Administrators invite users who belong to another team to another tenant.
-* Starting with release 2018.40, administrators can invite a user to more than one tenant.   A user who has access to multiple tenants has to log in only once, and can then switch from one tenant to another. As a side effect, the changes to multi-tenancy also results in fine-grained access control and in improved query performance.
+* Administrators can invite a user to more than one tenant.
+* A user who has access to multiple tenants has to log in only once, and can then switch from one tenant to another.
 
-## How Administrators Set Up Multi-Tenancy
+## How to Set Up Multi-Tenancy
 
-To set up multi-tenancy, administrators follow these steps:
+To set up multi-tenancy, follow these steps:
 
-1. Decide on the multi-tenant mode (see below), that is, whether they want a sandbox or strict multi-tenant setup.
+1. Decide on the multi-tenancy mode (see below), that is, whether they want a sandbox or strict multi-tenant setup.
 1. Request a multi-tenant setup from Wavefront with the following information:
    * Name of the tenants to create (one tenant per team).
    * Email addresses of the administrators of reach team.
```

Comment:
Email addresses of the administrators of reach team.
----> Email addresses of the administrators of each team.